workflow-templates/check-license.yml

```diff
@@ -1,12 +1,6 @@
 # Source: https://github.com/arduino/tooling-project-assets/blob/main/workflow-templates/check-license.md
-name: Check License
 
-env:
-  # TODO: Define the project's license file name here:
-  EXPECTED_LICENSE_FILENAME: LICENSE.txt
-  # SPDX identifier: https://spdx.org/licenses/
-  # TODO: Define the project's license type here
-  EXPECTED_LICENSE_TYPE: AGPL-3.0
+name: Check License
```

Comment:
You introduced a pointless blank line at line 2. Please remove it.

Reply:
Removed ✅